google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -206,8 +264,96 @@ def _text_from_layout(layout: documentai.Document.Page.Layout, text: str) -> str
     return result_text
 
 
-def _get_blocks(blocks: List[documentai.Document.Page.Block], text: str) -> List[Block]:
-    r"""Returns a list of Block.
+def _get_tokens_in_line(
+    line: documentai.Document.Page.Line, tokens: List[Token]
+) -> List[Token]:
+    r"""Returns a list of wrapped tokens inside line.
+
+    Args:
+        line (documentai.Document.Page.Line):
+            Required. A line in a page.
+        tokens (List[Token]):
+            Required. List of wrapped tokens.
+
+    Returns:
+        List[Token]:
+            A list of wrapped tokens that are inside a line.
+    """
+    start_index = line.layout.text_anchor.text_segments[0].start_index
+    end_index = line.layout.text_anchor.text_segments[0].end_index
+
+    return [
+        token
+        for token in tokens
+        if token.documentai_token.layout.text_anchor.text_segments[0].start_index
+        >= start_index
+        if token.documentai_token.layout.text_anchor.text_segments[0].end_index
+        <= end_index
+    ]
+
+
+def _get_lines_in_paragraph(
+    paragraph: documentai.Document.Page.Paragraph, lines: List[Line]
+) -> List[Paragraph]:
```

Comment:
This annotation of return type seems incorrect.

Reply:
Correct.